extraction/extractor.py

```diff
@@ -72,13 +72,13 @@ def __extract_data_from_html_head(self, soup_head):
     def __extract_data_from_html_body(self, node):
         """This function iterates over dom using dfs"""
 
-        is_content, _content = self.__is_content(node)
+        _content = self.__is_content(node)
```

Comment:
`__is_content(node)` to `__validate_and_extract_content(node)`

Reply:
Updated!